<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ederson Moura &lt;ederson.moura@resource.com.br&gt;</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATM Test Devices</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/res/images/app-icon.png</normaloff>:/res/images/app-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover, QToolButton:hover {
    background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop : 1.0 #fedb74);
}

QTabWidget QWidget {
   background: none;
}

QTabWidget::pane {
    border: 0px;
    top: 0.1em;
    border-width: 1px;
    border-style: solid;
    border-color: grey;
}

QTabBar::tab {
    width: 150px;
    margin-left: 0.5;
    margin-right: 0.5;
    border: 1px solid #7a7a7a;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTreeView {
    background: none;
    color: black;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QTreeView::item:selected:active {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
    border-width: 0px;
}

QTreeView::item:selected {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
    border-width: 0px;
    color: white;
}

QTreeView::item:hover {
    background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop : 1.0 #fedb74);
}


QComboBox {
	border: 1px solid #333333;
	border-radius: 3px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
	padding: 1px 23px 1px 3px;
	min-width: 6em;
    color: white;
    font: bold large &quot;Arial&quot;;
    height: 30px;
}

QComboBox:hover {
     text-decoration: underline;
     background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop :   1.0 #fedb74);
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 20px;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
     image: url(:res/images/combo-arrow-down.png);
}

QComboBox QAbstractItemView {
    background-color: #4f4f4f;
    color: white;
   selection-background-color: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa, stop : 0.5 #ffbb6e, stop : 0.55 #feae42, stop : 1.0 #fedb74);
}
 
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidgetDevices">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidgetMain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionWizardMenu"/>
    <addaction name="actionShowReportLogMenu"/>
    <addaction name="separator"/>
    <addaction name="actionCloseMenu"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutMenu"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionWizardBar"/>
   <addaction name="actionShowReportLogBar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWizardBar">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/res/images/wizard-select.png</normaloff>:/res/images/wizard-select.png</iconset>
   </property>
   <property name="text">
    <string>Run Tests</string>
   </property>
   <property name="toolTip">
    <string>Run the automated tests</string>
   </property>
   <property name="statusTip">
    <string>Run the automated tests</string>
   </property>
   <property name="whatsThis">
    <string>Run the automated tests</string>
   </property>
  </action>
  <action name="actionCloseMenu">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutMenu">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionShowReportLogBar">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/res/images/report-icon.png</normaloff>:/res/images/report-icon.png</iconset>
   </property>
   <property name="text">
    <string>Report test</string>
   </property>
   <property name="toolTip">
    <string>Report containing the test data</string>
   </property>
   <property name="statusTip">
    <string>Report containing the test data</string>
   </property>
   <property name="whatsThis">
    <string>Report containing the test data</string>
   </property>
  </action>
  <action name="actionShowReportLogMenu">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/res/images/report-icon.png</normaloff>:/res/images/report-icon.png</iconset>
   </property>
   <property name="text">
    <string>Report test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionWizardMenu">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/res/images/wizard-select.png</normaloff>:/res/images/wizard-select.png</iconset>
   </property>
   <property name="text">
    <string>Run Tests</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
